src/main/resources/js/main.js

```diff
@@ -577,4 +493,4 @@ console.log(`NODE_ENV: ${process.env.NODE_ENV}`);
 ReactDOM.render(
   <FullPage />,
   document.getElementById('content')
-);
+);
```

Comment:
I'd think files should probably end with a newline. See, for example, http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline. If you'd rather, I could accept this and we could fix it up as part of #48.